common/src/service/ResourceHelper.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
There are two copies of this file, one here and another at iothub/device/src/Common/ResourceHelper.cs. Is that intentional?